### PR TITLE
Support for multiple links. Able to cast to screen, but need way to

### PR DESCRIPTION
**Pull request by leto-II**
extract .mp4 from links.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
             'pychromecast',
             'cast',
             'docopt',
+            'beautifulsouyp4',
```
Comment (dymurray) [inline setup.py:13]: typo?

File: jacarutu.py [hunk 2]
```diff
@@ -18,17 +23,33 @@ def main():
         if args.episode:
             find_episode(args.search_string, args.season, args.episode)
         else:
-            print search(args.search_string)
+            x=2#print search(args.search_string)
```
Comment (dymurray) [inline jacarutu.py:26]: Why is this still here?

**Comment by dymurray** [comment]
Closing this, clean it up in your branch first then we can submit a new one.

**Comment by leto-II** [comment]
Ya my b

**Pull request closed without merging**